modules/dreamview/frontend/src/renderer/obstacles.js

```diff
@@ -132,6 +129,49 @@ export default class PerceptionObstacles {
                 iconIdx++;
             }
         }
+    }
+
+    updateSensorMeasurements(world, coordinates, scene, isBirdView) {
+        const sensorMeasures = world.sensorMeasurements;
+        if (!STORE.options['showObstaclesLidarSensor'] && !STORE.options['showObstaclesRadarSensor']
+            && !STORE.options['showObstaclesCameraSensor']) {
+            return;
+        }
+        for (const key in sensorMeasures) {
+            const sensorType = this.deduceSensorType(key.toLowerCase());
+            if (!sensorType || !STORE.options['showObstacles' + sensorType]) {
+                continue;
+            }
+            for (const value of sensorMeasures[key]["sensorMeasurement"]) {
```

Comment:
If you'd like to shorten the variable name, "measurement" might be better than "sensorMeasure"

for (cost measurement of ....) and remove line 146.